doc/api/stream.md

```diff
@@ -1103,6 +1103,16 @@ added: v12.3.0
 
 Getter for the property `objectMode` of a given `Readable` stream.
 
+##### readable.readCount
+<!-- YAML
+added: CHANGEME
+-->
+
+* {number}
+
+Number of time `data` has been called in `objectMode`, otherwise number of bytes
```

Comment:
I think this needs to be reworded because you can also read data via `.read()`, not just via `'data'` events.